src/core/electrostatics_magnetostatics/mmm2d.cpp

```diff
@@ -195,11 +189,11 @@ static int n_localpart = 0;
 /** temporary buffers for product decomposition */
 static std::vector<double> partblk;
 /** for all local cells including ghosts */
-static std::vector<double> lclcblk;
+static double *lclcblk = nullptr;
```

Comment:
What is the advantage of lclcblk being a double* ?

Reply:
That his would merge. :-) I'll re-add the vectors...